activerecord/test/cases/adapters/postgresql/json_test.rb

```diff
@@ -33,6 +43,14 @@ def test_deserialize_with_array
     x.reload
     assert_equal ["foo" => "bar"], x.objects
   end
+
+  def test_pretty_print
```

Comment:
Can you move this test into `JSONSharedTestCases`? `json` type is supported not only for PostgreSQL but also MySQL etc.